### PR TITLE
feature/pr_template

### PR DESCRIPTION
**Pull request by malloryprow**
## Pull Request Testing ##

1. Updates PR template
2. Addes fix directory to .gitignore

- [ ] Describe testing already performed for this Pull Request:</br>
> N/A

- [ ] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>
> N/A

- [ ] Has the code been checked to ensure that no errors occur during the execution? **N/A**

- [ ] Do these updates/additions include sufficient testing updates? **N/A**

- [ ] Please complete this pull request review by **8/20/2024**.</br>

## Pull Request Checklist ##

- [ ] Review the source issue metadata (required labels, projects, and milestone).
- [ ] Complete the PR description above.
- [ ] Ensure the PR title matches the feature branch name.
- [ ] Check the following:
- [ ]  Instructions provided on how to run
- [ ]  Developer's name is replaced by ${user} where necessary throughout the code
- [ ]  Check that the ecf file has all the proper definitions of variables
- [ ]  Check that the jobs file has all the proper settings of COMIN and COMOUT and other input variables
- [ ]  Check to see that the output directory structure is followed
- [ ]  Be sure that you are not using MET utilities outside the METplus wrapper structure

- [ ] After submitting the PR, select **Development** issue with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,28 +1,24 @@
...
-- [ ]  Be sure that you are not using MET utilities outside the METplus wrapper structure
-
-- [ ] After submitting the PR, select **Development** issue with the original issue number.
-- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
-- [ ] Close the linked issue.
+<b>Note to developers: You must use this PR template!</b>
+
+## Description of Changes
+
+> Please include a summary of the changes and the related GitHub issue(s). Please also include relevant motivation and context.
```
Comment (AndrewBenjamin-NOAA) [inline .github/pull_request_template.md:5]: I like that devs will start linking to the relevant github issues. That makes tracking much easier

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,28 +1,24 @@
...
+- [ ] Developer's name is removed throughout the code and have used `${USER}` where necessary throughout the code.
+- [ ] References the feature branch for `HOMEevs` are removed from the code.
+- [ ] J-Job environment variables, COMIN and COMOUT directories, and output follow what has been [defined](https://docs.google.com/document/d/1JWg_4q80aYmmAoD21GFjp9R9y5-3w7WGM3-0HJk0Pjs/edit#heading=h.7ysbr191vzu4) for EVS.
+- [ ] Jobs over 15 minutes in runtime have restart capability.
+- [ ] If applicable, changes in the `dev/drivers/scripts` or `dev/modulefiles` have been made in the corresponding `ecf/scripts` and `ecf/defs/evs-nco.def`? 
+- [ ] Jobs contain the approriate file checking and don't run METplus for any missing data.
+- [ ] Code is using METplus wrappers structure and not calling MET executables directly.
+- [ ] Log is free of any ERORRS or WARNINGS.
+
+## Testing Instructions
```
Comment (PerryShafran-NOAA) [inline .github/pull_request_template.md:22]: @malloryprow Please correct the spelling of ERRORs.

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -1,28 +1,24 @@
...
+<b>Note to developers: You must use this PR template!</b>
+
+## Description of Changes
+
+> Please include a summary of the changes and the related GitHub issue(s). Please also include relevant motivation and context.
+
+## Developer Questions and Checklist
+* Is this a high priorty PR? If so, why and is there a date it needs to be merged by?
+* Do you have any planned upcoming annual leave/PTO?
+* Are there any changes needed for when the jobs are supposed to run?
```
Comment (PerryShafran-NOAA) [inline .github/pull_request_template.md:10]: Who determines whether a PR is "high priority"?  Do we have a definition of high priority?

**Review by AndrewBenjamin-NOAA — APPROVED** [review]
I think these are good changes. My only comment is with regards to linking/closing issues being removed from the template. Not sure if it's appropriate to put in the PR template, but I think it's worth communicating to devs so that everything is documented. It was very easy to follow DTC's metplus issues and the PR's that fixed them when we were going through the bugzillas.

File: .github/pull_request_template.md (hunk 2 above)
Comment (malloryprow) [reply .github/pull_request_template.md:22]: Done

File: .github/pull_request_template.md (hunk 3 above)
Comment (malloryprow) [reply .github/pull_request_template.md:10]: No concrete definition but would be things like bug fixes for ops, correct failures happening in the parallel, etc.

Comment (PerryShafran-NOAA) [reply .github/pull_request_template.md:10]: OK good enough.  I'm not entirely sure how developers would answer this unless we tell them it's a priority fix.  For now, let's assume that it does mean for bugfixes and production/parallel failures.

File: .github/pull_request_template.md (hunk 2 above)
Comment (PerryShafran-NOAA) [reply .github/pull_request_template.md:22]: @malloryprow Once more please - it's two Rs before one R.

Comment (malloryprow) [reply .github/pull_request_template.md:22]: Oh sorry missed that. Hold on.

**Review by PerryShafran-NOAA — APPROVED** [review]
Everything looks good and is approved for merge.

**Pull request merged**